s3cmd

```diff
@@ -2171,6 +2171,35 @@ def cmd_sync(args):
         return cmd_sync_remote2remote(args)
     raise ParameterError("Invalid source/destination: '%s'" % "' '".join(args))
 
+def cmd_getacl(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    if len(args) != 1:
+        raise ParameterError("Too few parameters! Expected: <bucket>")
+    if not S3Uri(args[0]).has_bucket():
+        raise ParameterError("Invalid bucket name: '%s'" % args[0])
+    uri = S3Uri(args[0])
+    try:
+        acl = s3.get_acl(uri)
+        acl_grant_list = acl.getGrantList()
+        owner = acl.getOwner()
+        output(u"Bucket:     %s" % uri.bucket())
+        output(u"Owner:      %s (ID: %s)" % (owner['nick'], owner['id']))
```

Comment:
Good idea to show the owner, but you will have to do a special for "nick".
Based on what is on the following page, it will not be reported anymore by aws:
https://docs.aws.amazon.com/AmazonS3/latest/API/API_Grantee.html

I already don't see it.
So you could do something to only display it when it is available, but I think that it will be simpler and safer just to remove it and just display the id.